db/manager.go

```diff
@@ -187,16 +187,17 @@ func (m *Manager) Close() error {
 	return m.config.Datastore.Close()
 }
 
-// getDBConfig copies the manager's base config and
-// wraps the datastore with an id prefix.
-func getDBConfig(id thread.ID, base *Config) *Config {
+// getDBConfig copies the manager's base config,
+// wraps the datastore with an id prefix,
+// and merges specified collection configs with those from base
+func getDBConfig(id thread.ID, base *Config, collections ...CollectionConfig) *Config {
 	return &Config{
 		RepoPath: base.RepoPath,
 		Datastore: wrapTxnDatastore(base.Datastore, kt.PrefixTransform{
 			Prefix: dsDBManagerBaseKey.ChildString(id.String()),
 		}),
-		EventCodec: base.EventCodec,
-		JsonMode:   base.JsonMode,
-		Debug:      base.Debug,
+		EventCodec:  base.EventCodec,
+		Debug:       base.Debug,
+		Collections: append(base.Collections, collections...),
```

Comment:
We combine the default collections that the manager was created with (using `Option`s) with any default collections specified in the call to `manager.NewDB` or `manager.NewDBFromAddress`.

Reply:
Nice